assets/page/SettingsAccount.svelte

```diff
@@ -30,6 +31,7 @@ updateUserOp.on('start', req => {
   if (!req.body.password) delete req.body.password;
   req.body.highlight_keywords = req.body.highlight_keywords.split(/[.,\s]+/).map(str => str.trim());
   user.update({highlightKeywords: req.body.highlight_keywords});
+  user.update({ignoreStatuses: user.ignoreStatuses});
```

Comment:
I think this is a bug. Should probably just be `user.update({ignoreStatuses})`. You can also do this: `user.update({highlightKeywords: req.body.highlight_keywords, ignoreStatuses});` because of the magical `{ignoreStatuses} == {ignoreStatuses: ignoreStatuses}` syntax in es2015.

Reply:
Oh. Strike the previous comment. Just get rid of this, since you got it covered inside `updateUserFromForm()`.